packages/Webkul/Product/src/Type/Configurable.php

```diff
@@ -96,7 +96,7 @@ public function update(array $data, $id, $attribute = "id")
     {
         $product = parent::update($data, $id, $attribute);
 
-        if (request()->route()->getName() != 'admin.catalog.products.massupdate') {
+        if (!request()->route() || request()->route()->getName() != 'admin.catalog.products.massupdate') {
```

Comment:
Not like that, check this,
https://github.com/bagisto/bagisto/blob/2b4ca622b43910536b8441fce6a5e5f66a221774/packages/Webkul/Product/src/Type/AbstractType.php#L212

~~~php
$route = request()->route() ? request()->route()->getName() : "";

if ($route !== 'admin.catalog.products.massupdate') {
~~~